docs/events.md

```diff
@@ -0,0 +1,28 @@
+# Events
+
+Events that may be of interest to users of the Assisted Service are made available via the REST API.  The events are stored in the SQL database.  Each event is associated with a cluster and has a severity; some events are also associated with a host.
+
+Using events, a user should be able to understand how the cluster reached its current state, and understand what, if anything, is wrong with it.
+
+## Event emission guidelines
+
+### ERROR
+Something wrong has happened that must be investigated.  These should be relatively rare and are important to understand, so make them as verbose as possible - describe what happened, what the user might do to mitigate it, and anything needed to debug it.
+
+Examples:
+1. REST API call failure.
```

Comment:
I think it's weird listing only one item. I would have add some more examples, or put it in a sentence

Reply:
Added more examples